velox/exec/tests/MultiFragmentTest.cpp

```diff
@@ -935,3 +935,30 @@ TEST_F(MultiFragmentTest, exchangeDestruction) {
   leafTask = nullptr;
   rootTask = nullptr;
 }
+
+TEST_F(MultiFragmentTest, cancelledExchange) {
+  // Create a source fragment borrow the output type from it.
+  auto planFragment = exec::test::PlanBuilder()
+                          .tableScan(rowType_)
+                          .filter("c0 % 5 = 1")
+                          .partitionedOutput({}, 1, {"c0", "c1"})
+                          .planFragment();
+
+  // Create task with exchange.
+  auto planFragmentWithExchange =
+      exec::test::PlanBuilder()
+          .exchange(planFragment.planNode->outputType())
+          .partitionedOutput({}, 1)
+          .planFragment();
+  auto exchangeTask =
+      makeTask("output.0.0.1", planFragmentWithExchange.planNode, 0);
+  // Start the task and abort it straight away.
+  Task::start(exchangeTask, 2);
+  exchangeTask->requestAbort();
+
+  /* sleep override */
```

Comment:
Remove sleep override comment?

Reply:
It is actually needed to avoid lint warning, but it must be right before the sleep statement, which is, sadly, not the case here.